lib/ansible/modules/clustering/cloudera/cloudera_hdfs_ha.py

```diff
@@ -0,0 +1,187 @@
+        entry = { 'jnHostId': host_id, 'jnName': journal_node['journal_role_name'] }
+        journal_nodes_info.append(entry)
+    return journal_nodes_info
+
+def enable_hdfs_ha(cloudera, connection, cluster_name, active_role_name, standby_host_role_type, nameservice, jns, standby_role_name):
+    cluster = connection.get_cluster(cluster_name)
+    service = cloudera.get_service_by_type(cluster, 'HDFS')
+    standby_host_id = cloudera.get_hostid_by_role_type(service, standby_host_role_type)[0]
+    zookeeper_service_name = cloudera.get_service_by_type(cluster, 'ZOOKEEPER').name
+    journal_nodes_info = compose_journal_nodes_info(cloudera, connection, jns)
+    service.enable_nn_ha(
+        active_name=active_role_name, standby_host_id=standby_host_id, nameservice=nameservice, jns=journal_nodes_info,
+        standby_name=standby_role_name, zk_service_name=zookeeper_service_name
+    ).wait()
+    return True
+
+def main():
+    module = AnsibleModule(
+        argument_spec=dict(
+            host=dict(required=True, type='str'),
+            port=dict(default=7180, type='int'),
+            username=dict(default='admin', type='str'),
+            password=dict(default='admin', no_log=True, type='str'),
+            api_version=dict(default=12, type='int'),
+            use_tls=dict(default=False, type='bool'),
+            cluster_name=dict(required=True, type='str'),
+            active_role_name=dict(default='NAMENODE', type='str'),
+            standby_host_role_type=dict(default='SECONDARYNAMENODE', type='str'),
+            nameservice=dict(default='HDFS-HA-NAMESERVICE', type='str'),
+            jns=dict(default=None, type='list'),
+            standby_role_name=dict(default='NAMENODE-STANDBY', type='str'),
+            state=dict(default='enable_hdfs_ha', choices=['enable_hdfs_ha'])
+        ),
+        supports_check_mode=True
+    )
+
+    host = module.params['host']
+    port = module.params['port']
+    username = module.params['username']
+    password = module.params['password']
+    api_version = module.params['api_version']
+    use_tls = module.params['use_tls']
+    cluster_name = module.params['cluster_name']
+    active_role_name = module.params['active_role_name']
+    standby_host_role_type = module.params['standby_host_role_type']
+    nameservice = module.params['nameservice']
+    jns = module.params['jns']
+    standby_role_name = module.params['standby_role_name']
+    state = module.params['state']
+    changed=False
+
+    cloudera = Cloudera(module, host, port, username, password, api_version, use_tls)
+    connection = cloudera.connect()
+
+    if module.check_mode:
+        module.exit_json(changed=True)
+
+    if state == 'enable_hdfs_ha':
+        changed = enable_hdfs_ha(cloudera, connection, cluster_name, active_role_name, standby_host_role_type, nameservice, jns, standby_role_name)
+
+    module.exit_json(changed=changed)
+
+if __name__ == '__main__':
+    from ansible.module_utils.basic import *
```

Comment:
Generally the imports go before the "if main" 
Only import what you need, rather than `*`